src/_includes/head.html

```diff
@@ -14,6 +14,15 @@
  <meta name="robots" content="{{ page.metarobots }}" />
 {% endif %}
 
+<!-- Google Tag Manager -->
+<script>
+    (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-WH8CK39');
+</script>
+<!-- End Google Tag Manager -->
```

Comment:
Let's not add the comments?

Reply:
@LL782 extending this, do you think we should remove all the comments on the head? there are quite a few of them.